### ISSUE TITLE
Feature to specify cop that cannot be disabled

### ISSUE THREAD

**Issue by ydah** [rubocop/rubocop#10782]
## Is your feature request related to a problem? Please describe.

If you are developing a large rails application, you will be working with many developers.
In such cases, there are certain things that we want the application as a whole to be compliant with, but currently there is a loophole that exists through the use of comments to disable them.

It’s difficult to watch through all the changes to make sure we are not disabling a compliant cop as an overall rule, and I would be happy to have a solution to this.

## Describe the solution you'd like

For example, the following rules that cannot be disabled can be specified in `.rubocop.yml`.

```yml
Style/For:
  Force: enabled
```

## Describe alternatives you've considered

I would be happy to be able to specify the cop I want to correct, even if it is not this method.

**Comment by jonas054** [comment]
There is a cop with this default configuration:

```yaml
Style/DisableCopsWithinSourceCodeDirective:
  Description: >-
                 Forbids disabling/enabling cops within source code.
  Enabled: false
  VersionAdded: '0.82'
  VersionChanged: '1.9'
  AllowedCops: []
```

I think you can use it to achieve what you describe here. Right?

**Comment by ydah** [comment]
This certainly seems to solve what I want to do. Thank you so much!
